<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_water_meter_display"/>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_report"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionHelpTools"/>
   </widget>
   <widget class="QMenu" name="menuCOnfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionAddWaterDevice"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuCOnfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSyncView"/>
   <addaction name="actionUpdateDeviceStatus"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelpTools">
   <property name="text">
    <string>HelpTools</string>
   </property>
  </action>
  <action name="actionAddWaterDevice">
   <property name="text">
    <string>AddWaterDevice</string>
   </property>
  </action>
  <action name="actionUpdateDeviceStatus">
   <property name="text">
    <string>UpdateDeviceStatus</string>
   </property>
  </action>
  <action name="actionSyncView">
   <property name="text">
    <string>SyncView</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
